gamebook/fight_scene.py

```diff
@@ -0,0 +1,45 @@
+from gamebook.scene import Scene
+import gamebook.important_strings
+
+
+class FightScene(Scene):
+    """
+        A scene containing a battle with a monster
+
+        arguments:
+
+        next_scenes - a list of two NextScene objects, 1st is the victory scene
+        and 2nd is the the defeat scene
+
+        enemy - a monster instance
+
+    """
+    def __init__(self, name, desc, next_scenes, enemy):
+        self.name = name
+        self.desc = desc
+        self.next_scenes = {"win": next_scenes[0], "loss": next_scenes[1]}
+        self.enemy = enemy
+
+    def get_name(self):
+        return super().get_name()
+
+    def show_desc(self):
+        return super().show_desc()
+
+    def show_options(self):
+        return super().show_options()
+
+    def show_monster(self):
+        data_string = ""
+        data_string += self.enemy.name + "\n"
+        for trait in self.enemy.traits:
+            data_string + f"trait name: {trait.name}, value: {trait.value}\n"
+
+        return data_string
+
+    def run_fight(self, hero):
+        if hero.fight(gamebook.important_strings.power_trait_name,
```

Comment:
why are those the parameters for `fight`? why isn't it `hero.fight(monster)` and inside it uses the traits?

Reply:
i need a way to define which traits to use for attack and health... should i turn the trait list to a dict?
im imagining something like this:
`{"attack": power_trait, "health":health_trait, "luck": luck_trait}`

Reply:
Sounds better yeah.